src/main/java/page_objects/BookTicketPage.java

```diff
@@ -46,4 +46,12 @@ public Ticket getSuccessfulTicket() {
 
         return newTicket;
     }
+
+    public boolean isDepartStationSelected(String departStation) {
+        return drnDepartFrom.getSelectedOptionAsText().equals(departStation);
```

Comment:
you should not write function like this, because it's harder to check if something went wrong. 
just use getSelectedDepartStation and compare it with expected departStation using assert equals.
that way if something goes wrong, you can see what is expected and what is actual on test result